jujugui/static/gui/src/app/app.js

```diff
@@ -1157,6 +1157,33 @@ YUI.add('juju-gui', function(Y) {
       this.navigate(url);
     },
 
+    /** Chooses an env to connect to from the env list based on config.
+
+      @method _pickEnv
+      @param {String} envList The list of environments to pick from.
+      @return {Object} The selected environment.
```

Comment:
I dunno; I think mixed terminology has hurt us in discussions, and the objects etc are all "env" something or other, so referring to it otherwise seems more confusing to me.